tic-tac-toe/src/components/board/board.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Square from '../square/square';
+import './board.css';
+
+class Board extends Component {
```

Comment:
Why are you using class if don't have a state?

Reply:
Now I use functional component.
```suggestion
function Board({squares, onClick}) {
```